scripts/in_container/prod/clean-logs.sh

```diff
@@ -17,14 +16,14 @@
 # specific language governing permissions and limitations
 # under the License.
 
-set -euo pipefail
+set -e pipefail
```

Comment:
Why?

Reply:
Yeah... why ?

Reply:
It won't work by the way :)

Reply:
I reverted by changes and pushed by code again. I was missing pre-commit test in my first push.